<launch>
  <arg name="access_frequency" default="1.0" />
  <arg name="chip_name" default="gpiochip0" />
  <arg name="active_low" default="false" />

  <group ns="dio">
    <rosparam file="$(find dio_ros_driver)/config/port_list.yaml" />
    <node name="dio_ros_driver" type="dio_ros_driver_node" pkg="dio_ros_driver">
      <param name="access_frequency" type="double" value="$(arg access_frequency)" />
      <param name="chip_name" type="string" value="$(arg chip_name)" />
      <param name="active_low" type="bool" value="$(arg active_low)" />
    </node>
  </group>
</launch>